src/LiquidationPair.sol

```diff
@@ -340,17 +344,22 @@ contract LiquidationPair is ILiquidationPair {
   }
 
   /// @notice Returns the current auction start time
+  /// @dev If the first period has not started yet, this will return the first period start time.
   /// @return The start timestamp
   function getPeriodStart() external returns (uint256) {
     _checkUpdateAuction();
-    return _getPeriodStart(_computePeriod());
+    uint256 currentPeriod = _computePeriod();
+    return currentPeriod == 0 ? firstPeriodStartsAt : _getPeriodStart(currentPeriod);
```

Comment:
Should be handled in `_getPeriodStart()`, this way we don't have to implement the logic again in `getPeriodEnd`.

Reply:
Period `0` doesn't have a start date since it's not a valid period. These functions check if the period is zero, and return the first *valid* period start/end instead.

Reply:
If we don't want to duplicate logic, we could make `getPeriodStart()` a public function and call it in `getPeriodEnd()`, but that would use more gas for `getPeriodEnd()` calls so it's a small trade off.